### ISSUE TITLE
Updating from the Foundry Interface is not working

### ISSUE THREAD

**Issue by Mermut** [BernhardPosselt/pf2e-kingmaker-tools#3]
I've noticed that updating from the Foundry interface is not getting the updates to this module.
Neither the Update All button nor clicking the 'update' button from the module specifically does it.

The only way to get the update is to copy/paste the manifest url into Foundry interface

**Comment by BernhardPosselt** [comment]
If you've installed it manually before (as in: extracting it into your modules folder) you don' get auto updates. You need to remove it completely and install it from the registry again.

At least that's my guess, otherwise I'd recommend asking on the Foundry Discord.

**Comment by Mermut** [comment]
I've always done it from Foundry... I also tried uninstalling and reinstalling (from Foundry).
*small shrug* Pasting the manifest URL into Foundry works.. just means I have to manually check for updates, which is how I noticed in the first place, actually. I saw an update via the module add-on page and wondered why it wasn't showing up when I used the update all (or specific update) button on the Foundry (not Forge) side.
